src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -1,128 +1,292 @@
 
-  <div class="filters clearfix">
-    <button style="width: 15%" class="filter-button" data-scope="aws-console">AWS Console</button>
-    <button style="width: 15%" class="filter-button" data-scope="aws-cli">AWS CLI</button>
-  </div>
+</section>
+
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+<div class="filters clearfix">
+  <button style="width: 15%" class="filter-button" data-scope="aws-console">AWS Console</button>
+  <button style="width: 15%" class="filter-button" data-scope="aws-cli">AWS CLI</button>
+</div>
 
 <section class="filter-content" markdown="1" data-scope="aws-console">
 
-1.  Click **Endpoints** in the sidebar.
-1.  Click **Create Endpoint**.
-1.  On the **Create Endpoint** page, for the **Service Category** field, select **Find service by name**.
-1.  In the **Service Name** field, enter the **Service Name** copied from the connection modal in [Step 1](#step-1).
-1.  Click **Verify**.
-1.  In the **VPC** field, enter the ID of the VPC you want to create your endpoint in.
-1.  Verify that the subnets are pre-populated.
-1.  In the **Security group** section, select the security group you created in [Step 8](#step-8) and uncheck the box for **default** security group.
-1.  Click **Create Endpoint**.
+1. Click **Endpoints** in the sidebar.
+1. Click **Create Endpoint**.
+1. On the **Create Endpoint** page, for the **Service Category** field, select **Find service by name**.
+1. In the **Service Name** field, enter the **Service Name** copied from the connection modal in [Step 1](#step-1).
+1. Click **Verify**.
+1. In the **VPC** field, enter the ID of the VPC you want to create your endpoint in.
+1. Verify that the subnets are pre-populated.
+1. In the **Security group** section, select the security group you created in [Step 8](#step-8) and uncheck the box for **default** security group.
+1. Click **Create Endpoint**.
 
-      The VPC Endpoint ID displays.
+    The VPC Endpoint ID displays.
 
-1.  Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
+1. Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
 
-  </section>
+</section>
 
-  <section class="filter-content" markdown="1" data-scope="aws-cli">
+<section class="filter-content" markdown="1" data-scope="aws-cli">
 
-1.  Substitute the values from the previous steps and run the following AWS CLI command:
+1. Substitute the values from the previous steps and run the following AWS CLI command:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ aws ec2 create-vpc-endpoint --region $REGION \
-    --vpc-id $VPC_ID --subnet-ids $SUBNET_ID1 $SUBNET_ID2 \
-    --vpc-endpoint-type Interface --security-group-ids \
-    $SECURITY_GROUP_ID1 $SECURITY_GROUP_ID2 --service-name \
-    $SERVICE_NAME_PROVIDED_BY_COCKROACH
+    aws ec2 create-vpc-endpoint --region {REGION} \
+                                  --vpc-id {VPC_ID} \
```

Comment:
@nickvigilante I think a tooling bug might be requiring these extra two spaces on hanging indent. WDYT? I've verified that there are no tab characters in the file.